snowpack/src/index.ts

```diff
@@ -75,13 +76,14 @@ export async function cli(args: string[]) {
   // TODO: process.cwd() okay here? We should remove this requirement on a package.json for v3.0.
   let pkgManifest: any;
   try {
-    pkgManifest = require(path.join(process.cwd(), 'package.json'));
+    pkgManifest = await fs.readFile(path.join(process.cwd(), 'package.json'), 'utf8');
```

Comment:
Something I recently learned: `utf8` is the proper format, not `utf-8`. I forget which Node API it is (something with Buffers, I think), but one throws an error on `utf-8`. I think `readFile()` accepts both, so it’s the same. Anyway, something I’m starting to look out for in PRs is just `utf8` consistency.